addons/popochiu/editor/gizmos/plugin.cfg

```diff
@@ -0,0 +1,7 @@
+[plugin]
+
+name="GizmoClickable"
+description=""
+author=""
```

Comment:
Where are the names of the awesome guys who created such amazing plugin? I know this is not shown in the plugins list, but people will like to know about them when looking at the code.

Reply:
Done. Blame @stickgrinder for what's in there now.

Reply:
My bad, my bad!